core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/ingest/SpatialTemporalDimensionalityTypeProvider.java

```diff
@@ -36,7 +36,7 @@ public SpatialTemporalDimensionalityTypeProvider() {}
 
 	@Override
 	public String getDimensionalityTypeName() {
-		return "spatial-temporal";
+		return "spatial_temporal";
```

Comment:
Definitely want this to be a hyphen, we should probably figure out what I broke to make it not work.